### ISSUE TITLE
Crash on debug

### ISSUE THREAD

**Issue by vbif1** [defold/editor2-issues#2374]
### Expected behaviour
normal debugger start

### Actual behaviour
crash in editor

### Steps to reproduce
click on "run with debugger" on any project (event empty)

<hr/>

<table>
<tr><td>Build time</td><td>2018-12-18T15:04:58.078787</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>e6badbdfbe3d296da36f9f8e2e4848ffdfb9a971</td></tr>
<tr><td>Defold engine sha</td><td>8def4b50c9aa049670e698d7dff3dc00d77e42d5</td></tr>
<tr><td>Defold version</td><td>1.2.144</td></tr>
<tr><td>Error</td><td><a href='https://sentry.io/defold/editor2/?query=id%3A"e23b71bc11df45128c802656ca2dae38"'>e23b71bc11df45128c802656ca2dae38</a></td></tr>
<tr><td>GPU</td><td>AMD Radeon RX 560 OpenGL Engine</td></tr>
<tr><td>GPU Driver</td><td>2.1 ATI-2.4.9</td></tr>
<tr><td>Java version</td><td>1.8.0_102-b14</td></tr>
<tr><td>OS arch</td><td>x86_64</td></tr>
<tr><td>OS name</td><td>Mac OS X</td></tr>
<tr><td>OS version</td><td>10.14.2</td></tr>
<table>

**Comment by vbif1** [comment]
Yes, I have a nice message about debugger first, but after 2-4 seconds defold close all windows and stop responding. Firewall is turned off. No tight loop in game, its literally an empty desktop example.

**Comment by vbif1** [comment]
Yes, i have a nice message about debugger first, but after 2 or 4 seconds defold closes all windows and stop responding (need "force quit" to run it again). Firewall is turn off. No tight loop in game, its a empty desktop example.

**Comment by matgis-king** [comment]
Hi again! It looks like you're having the same issue as #2396. Please have a look at the discussion there to solve the problem of Defold quitting!
